COMPATIBILITY.md

```diff
@@ -87,7 +91,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `ec2`: known to work as is
```

Comment:
Well that is not the case. We just tested some version of the EC2 plugin with some version of Workflow at some point and it just worked without changes.